src/api/lambdas/bedrock-api-backend/run_groups/getRunGroup.js

```diff
@@ -1,6 +1,24 @@
 /* eslint-disable import/extensions */
 /* eslint-disable no-console */
 import { newClient, getInfo } from '../utilities/utilities.js';
+import pgErrorCodes from '../pgErrorCodes.js';
+
+async function getAssetInfo(client, idValue, name) {
+  // Querying database to get information. Function can be used multiple times per method
+  // if we need information from multiple tables
+  const sql = 'SELECT a.*, b.display_name FROM bedrock.etl a left join bedrock.assets b on a.asset_name = b.asset_name where run_group like $1';
```

Comment:
We'll need the asset ID (asset_name) for use on the frontend. In cases like these, assume that the display_name and ID will need to move together as a unit (e.g. for cross-linking to an asset, we'll need the ID in the link target as well as the display name in the link text)

Reply:
Oh, nevermind. The asset_name is coming from the bedrock.etl table in this query. Disregard!